Gradata/docs/specs/cloud-sync-and-pricing.md

```
@@ -0,0 +1,325 @@
+Returns: { "accepted": 42, "deduped": 3, "rejected": [] }
+```
+
+Client behavior:
+- Triggered on Stop hook or every 5min when events accumulated.
+- Pushes since `last_push_event_id` watermark.
+- Chunks of 500 events max per request.
+- On 429: exponential backoff; on 5xx: retry with same `event_id`s (server dedups by `event_id`).
+
+### 3.4 Pull (incremental)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)